chain-abci/src/main.rs

```diff
@@ -84,6 +85,21 @@ fn get_enclave_proxy(opts: &AbciOpt) -> MockClient {
     MockClient::new(get_network_id())
 }
 
+pub fn get_tendermint_filename() -> String {
+    match std::env::var("TENDERMINT_HOME") {
```

Comment:
I think the standard env variable used is `TM_HOME` or `TMHOME`: https://github.com/tendermint/tendermint/search?q=tm_home&unscoped_q=tm_home
```suggestion
    match std::env::var("TM_HOME") {
```

Reply:
It's `TMHOME`. Changed in suggestion.